app/src/main/res/values/strings.xml

```diff
@@ -5,4 +5,7 @@
         exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor
         in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
         sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt laborum.</string>
+  <string name="stub_md" translatable="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
No need for this since `tools:text="@tools:sample/lorem"`: https://developer.android.com/studio/write/tool-attributes#toolssample_resources

Reply:
Nice hack. Although this generates a new string each time and would make the documentation screenshots inconsistent.